examples/asr/conf/conformer/multilang/conformer_transducer_bpe_multilang.yaml

```diff
@@ -109,8 +109,8 @@ model:
     d_model: 512
 
     # Sub-sampling params
-    subsampling: striding # vggnet or striding
-    subsampling_factor: 4 # must be power of 2
+    subsampling: striding # vggnet, striding, stacking or stacking_norm
```

Comment:
dw_striding is an option .. but lets leave it for now. Too much work, we can update it in another PR/

Reply:
updated them for all conformer configs.